.github/workflows/reusable-create-java-contrib-pull-request.yml

```diff
@@ -0,0 +1,67 @@
+name: Reusable - Create Java contrib pull request
+
+on:
+  workflow_call:
+    inputs:
+      version:
+        type: string
+        required: true
+    secrets:
+      BOT_TOKEN:
+        required: true
+  # to help with partial release build failures
+  workflow_dispatch:
+    inputs:
+      version:
+        description: "Version"
+        required: true
+
+jobs:
+  create-java-contrib-pull-request:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          repository: opentelemetrybot/opentelemetry-java-contrib
+          # this is the personal access token used for "git push" below
+          token: ${{ secrets.BOT_TOKEN }}
+
+      - name: Initialize pull request branch
+        env:
+          VERSION: ${{ inputs.version }}
+        run: |
+          git remote add upstream https://github.com/open-telemetry/opentelemetry-java-contrib.git
+          git fetch upstream
+          git checkout -b update-opentelemetry-sdk-to-${VERSION} upstream/main
+
+      - name: Update version
+        env:
+          VERSION: ${{ inputs.version }}
+        run: |
+          sed -i "s/^val otelVersion = .*/val otelVersion = \"$VERSION\"/" dependencyManagement/build.gradle.kts
```

Comment:
Can we add a comment to https://github.com/open-telemetry/opentelemetry-java-contrib/blob/main/dependencyManagement/build.gradle.kts#L14 saying that this variable is being managed as a part of the instrumentation release build?